msr_rr_cfp.md

```diff
@@ -0,0 +1,83 @@
+# Title (required)
+Provide the working title of your study. It may be the same title that you submit for publication of your final manuscript, but it is not a requirement.
+* **Example**:: Should your family travel with you on the enterprise?
+* Subtitle (optional): Effect of accompanying families on the work habits of crew members
+
+# Authors (required)
+At this stage, we believe that an unblinded/single blind review is most productive
+
+# Structured Abstract (required)
+## Background/Context
+What is your research about? Why are you doing this research, why is it interesting?
+
+**Example**: The enterprise is the flag ship of the federation, and it allows families to travel onboard. However, there are no studies that evaluate how this affects the crew members”
+
+## Objective/Aim
+What exactly are you evaluating? What are the objects of the study?
+
+**Example**: We are studying whether the frequency of sick days, the work effectiveness and efficiency differ between science officers who bring their family with them, compared to science officers who are serving without their family
+
```

Comment:
Might this need to be a bit "beefier" for a RR? I.e. given that the specific RQs / hypothesis are in focus (and should not later be changed) and there is less to say about results (since there are little to no) it makes sense to expect more of the Objective/Aim section? Are there existing RR's from other fields we can use as models/examples?

Reply:
How much sense would it make for a RR to _not_ be hypothesis-based? 
Isn't this what would differentiate a RR from a NIER/short paper on "an idea/a proposed study"?

Reply:
Might be also too much, but maybe we can slightly rephrase it to make explicit that both confirmatory and exploratory types of studies (to name just two) are welcome? Maybe we can add a curiosity-driven study as one example for having hypothesis as an actual outcome of the study and one where we want to test certain hypothesis? If you agree, I would then propose to change the term "evaluate" to "study" or "investigate" (or something similarly vague).

Reply:
Given the NIER tracks, I suggest we direct curiosity-driven research to those tracks instead of as RR, at least for now. Thus RR would only handle confirmatory studies.

Reply:
I would find it a bit peculiar that we do not allow RRs for exploratory research. I don't want to be a party pooper but why don't we use a more general, and more "established" template, and allow reviewers the possibility to provide input without us being too prescriptive? https://github.com/crsh/prereg

Reply:
Yeah, to some extent we don't know what we will get, so might be easier to be "liberal in what we accept" to use Postel's law. 

However we have had some good comments about overlap with NIER tracks. Do you not see a conflict with NIER?

Reply:
Well, honestly, I don't care about NIER. I care more that we get this right. Disqualifying exploratory research seems wrong imho.

Reply:
fixed this in 46b307c10f68b2df1a374c